docs/commands/codeception.md

```diff
@@ -15,16 +15,16 @@ Run all the generated tests:
 vendor/bin/codecept run functional -c dev/tests/acceptance/codeception.yml
 ```
 
-Run all tests without the `<group value="skip"/>` [annotation][]:
+Run all tests without the `<group value="example"/>` [annotation][]:
```

Comment:
Wasn't `<group value="skip">` deprecated in 3.0.0 for implicit skipping? I changed it to a different value to avoid confusion. Please confirm if I need to revert this change.

Reply:
1) This document is for Codeception. Please check Codeception document

2) Regardless the behavior of group skip in Codeception, it does not make sense to change it to use `example` here, doesn't it?

Reply:
Reverted the change.